SDK.ipynb

```diff
@@ -78,9 +91,21 @@
    },
    "outputs": [],
    "source": [
-    "import globus_sdk\n",
+    "# fill this line in with the code that you got\n",
+    "auth_code = \"YOUR CODE HERE\"\n",
+    "\n",
+    "# and exchange it for a response object containing your token(s)\n",
+    "token_response = native_auth_client.oauth2_exchange_code_for_tokens(auth_code)\n",
```

Comment:
This raised `GlobusAPIError: (401, 'Error', '{"error":"invalid_grant"}')` for me the first time I tried it. I don't know if it was an intermittent bug in Auth or what, but then trying to back up to the previous block and re-click the link just made Auth spit out "code_challenge reuse detected" and refuse to continue. Re-generating the link and then going through the flow and then pasting a new code worked, but I wonder if we could do *something* to make it easier for the general population to recover (e.g. a `try/except` here, putting the calls that could break in their own cells w/ additional explanation).